spesifikasjoner/afpant/afpant-kjoepekontrakt/kjoepekontrakt-teknisk-beskrivelse.md

```diff
@@ -64,11 +64,11 @@ OG eksistere i ZIP-arkivet.
 
 ## Validering og ruting (meglersystem)
 Håndtering av meldingstype [KjoepekontraktforespoerselFraBank](#meldingstype-kjoepekontraktforespoerselfrabank):
-- Systemleverandør/meglersystem søker blant alle sine kunders oppdrag/oppgjør etter kjøpekontrakt.
-- OG utvalget av kjøpekontrakter avgrenses til:
+- Systemleverandør/meglersystem søker blant alle sine kunders oppdrag/oppgjør etter kjøpekontrakt og/eller strukturerte data.
```

Comment:
er vel ikke og eller strukturerte data, det er "strukturerte data og eller kjøpekontrakt"
Slik jeg har forstått det vil man alltid ha noen strukturerte data, men signert kontrakt kan mangle?

Reply:
@JonEgilH Hva om teksten ble slik:  "... etter strukturerte kjøpekontraktsdata, eventuelt med tilhørende signert kjøpekontrakt, OG utvalget avgrenses til:"

Reply:
Heisann, hadde desverre glemt å følge opp denne (merket det nå som jeg var inne og redigerte på feature branch for forutsetningsbrev) men jeg er også enig med forslaget til Arild her :) Pusher straks denne endringen på teksten.